pyqmc/dmc.py

```diff
@@ -292,68 +303,83 @@ def rundmc(
       weights: The final weights from this calculation
       
     """
-    # Restart
-    if hdf_file is not None:
-        with h5py.File(hdf_file, "a") as hdf:
-            if "configs" in hdf.keys():
-                stepoffset = hdf["step"][-1] + 1
-                configs.load_hdf(hdf)
-                weights = np.array(hdf["weights"])
-                if verbose:
-                    print("Restarted calculation")
+    # Restart from HDF file
+    if hdf_file is not None and os.path.isfile(hdf_file):
+        with h5py.File(hdf_file, "r") as hdf:
+            stepoffset = hdf["step"][-1] + 1
+            configs.load_hdf(hdf)
+            weights = np.array(hdf["weights"])
+            eref = hdf['eref'][-1]
+            esigma=hdf['esigma'][-1]
+            if verbose:
+                print("Restarted calculation")
+    else:
+        warmup = 2
```

Comment:
was this meant to be used two lines below, in en = df[ekey[0]+ekey[1]][2:]? right now it's never used